src/llm/legacy/servable.cpp

```diff
@@ -0,0 +1,158 @@
+//*****************************************************************************
+// Copyright 2025 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <memory>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include "../../logging.hpp"
+#include "../../status.hpp"
+
+#pragma warning(push)
+#pragma warning(disable : 4005 4309 6001 6385 6386 6326 6011 4005 4456 6246)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#include "mediapipe/framework/calculator_graph.h"
+#pragma GCC diagnostic pop
+#pragma warning(pop)
+
+#include "../../http_payload.hpp"
+#include "../../mediapipe_internal/mediapipe_utils.hpp"
+#include "../apis/openai_completions.hpp"
+#include "../text_processor.hpp"
+#include "servable.hpp"
+
+namespace ovms {
+
+static std::string wrapTextInServerSideEventMessage(const std::string& text) {
+    std::stringstream ss;
+    ss << "data: " << text << "\n\n";
+    return ss.str();
+}
```

Comment:
Don't we already have such methods in basic servable? Looks pretty common to me.